cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -55,6 +55,10 @@ type autoscalingGceClientV1 struct {
 	gceService *gce.Service
 
 	projectId string
+
+	// These can be overridden, e.g. for testing.
```

Comment:
They can't really be overridden for anything else given they're private fields in a struct :P